### PR TITLE
vscode-extensions.42crunch.vscode-openapi: 4.25.1 -> 4.25.3

### PR DESCRIPTION
**Pull request by benhiemer**
## Description of changes

Update vscode-extensions.42crunch.vscode-openapi: 4.25.1 -> 4.25.3

- Changelog: https://github.com/42Crunch/vscode-openapi/blob/master/CHANGELOG.md#version-4253-april-23-2024
- Comparing changes: https://github.com/42Crunch/vscode-openapi/compare/v4.25.1...v4.25.3

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/editors/vscode/extensions/default.nix [hunk 1]
```diff
@@ -90,8 +90,8 @@ let
         mktplcRef = {
           publisher = "42Crunch";
           name = "vscode-openapi";
-          version = "4.25.1";
-          sha256 = "+hKQUJp9c0oyhePFmQEXAqtqKL3fkQ1nhopUPnhRZc4=";
+          version = "4.25.3";
+          sha256 = "1kz/M2od2gLSFgqW6LsPHgtm+BwXA+0+7z3HyqNmsOg=";
```
Comment (superherointj) [inline pkgs/applications/editors/vscode/extensions/default.nix:94]:
```suggestion
          sha256 = "1kz/M2od2gLSFgqW6LsPHgtm+BwXA+0+7z3HyqNmsOg=";
```
Please convert to SRI / hash.

**Comment by superherointj** [comment]
I saw that you added the release notes / changelog to PR. The ideal location to add that is to the body of upgrade commit.

**Comment by superherointj** [comment]
@ofborg eval

**Comment by superherointj** [comment]
ping @benhiemer

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 307296` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

**Review by superherointj — APPROVED** [review]

**Comment by benhiemer** [comment]
> I saw that you added the release notes / changelog to PR. The ideal location to add that is to the body of upgrade commit.

Thanks for the advice, I will do it like this in the future. And thanks for applying your suggestion, I was busy the last days and could not do it. Sry for that.

**Comment by superherointj** [comment]
@ofborg eval

**Pull request merged**